docs/jwst/photom/reference_files.rst

```diff
@@ -31,38 +42,60 @@ dependent. The current row selection criteria are:
    - Fixed Slits: Filter, Grating, and Slit name
    - IFU and MOS: Filter and Grating
 
+AREA map Reference Files
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+For FGS, photom reference files are selected based on the values of INSTRUME and DETECTOR
+in the science data file.
+
+For MIRI photom reference files are selected based on the values of INSTRUME, DETECTOR, and EXP_TYPE
+in the science data file.
+
+For NIRCam, photom reference files are selected based on the values of INSTRUME, DETECTOR, and EXP_TYPE
+in the science data file.
+
+For NIRISS, photom reference files are selected based on the values of INSTRUME, DETECTOR, and EXP_TYPE
+in the science data file.
+
+For NIRSpec, photom reference files are selected based on the values of INSTRUME, DETECTOR, and EXP_TYPE
+in the science data file.
+
+Reference File Format
+---------------------
+
 PHOTOM Reference File Format
-----------------------------
-Photom reference files are FITS format with an empty primary data unit.
-The table-based photom files used for all instruments modes other than
-MIRI MRS have a single TABLE extension.
-The columns of the table vary with 
-instrument mode according to the selection criteria listed above. The first few
-columns always correspond to the row selection criteria, such as Filter and
-Pupil, or Filter and Grating. The remaining columns contain the data relevant
-to the photometric conversion and consist of PHOTMJSR, UNCERTAINTY, NELEM,
-WAVELENGTH, and RELRESPONSE. The table column names and data types are
-listed below.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Except for MIRI MRS, photom reference files are FITS format with a single
+BINTABLE extension.  The primary data unit is always empty.  The columns of
+the table vary with instrument according to the selection criteria listed 
+above. The first few columns always correspond to the selection criteria, such
+as Filter and Pupil, or Filter and Grating. The remaining columns contain the
+data relevant to the photometric conversion and consist of PHOTMJSR,
+UNCERTAINTY, NELEM, WAVELENGTH, and RELRESPONSE.  The table column names and
+data types are listed below.
+
 
 * FILTER (string) - MIRI, NIRCam, NIRISS, NIRSpec
 * PUPIL (string) - NIRCam, NIRISS
 * ORDER (integer) - NIRISS
 * GRATING (string) - NIRSpec
 * SLIT (string) - NIRSpec Fixed-Slit
 * SUBARRAY (string) - MIRI Imager/LRS
+* BAND (string) - MIRI MRS
```

Comment:
No, there isn't a BAND column for MIRI MRS. MIRI MRS mode uses a different format photom reference file that is not in the form of a table (it's a regular data array).